script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_1B.txt

```diff
@@ -180,7 +180,7 @@ Even if this person is alive and isn't human.
 Because in reality, I killed her with my very own hands.
 <Page58>
 "Come on, why are you all quiet again?
-What do you even me by 'I see', Shiki?"
+What do you even mean by 'I see', Shiki?"
```

Comment:
I need to re-enable flycheck for text buffers :aokoheh:

Reply:
Eight Megs And Constantly Spellchecking